pkg/ansible/runner/internal/inputdir/inputdir.go

```diff
@@ -60,6 +62,66 @@ func (i *InputDir) addFile(path string, content []byte) error {
 	return err
 }
 
+// copyFile copies a file from src to dst
```

Comment:
**Note**: copyFile and copyDir are both MIT licensed here: https://github.com/Depado/articles, either need to change implementation or reference license here.